roles/openshift_node/defaults/main.yml

```diff
@@ -68,6 +68,7 @@ default_r_openshift_node_image_prep_packages:
 - atomic
 - cifs-utils    # requested by Azure
 - samba-common  # requested by Azure
+- samba-client  # requested by Azure
```

Comment:
Was this required or was this for testing?

Reply:
I am _fairly_ sure it isn't needed, but the documentation has it and I think it might be useful for troubleshooting if needed.